internal/pkg/agent/cmd/install.go

```diff
@@ -49,6 +50,8 @@ would like the Agent to operate.
 	cmd.Flags().String(flagInstallBasePath, paths.DefaultBasePath, "The path where the Elastic Agent will be installed. It must be an absolute path.")
 	cmd.Flags().Bool(flagInstallUnprivileged, false, "Installed Elastic Agent will create an 'elastic-agent' user and run as that user. (experimental)")
 	_ = cmd.Flags().MarkHidden(flagInstallUnprivileged) // Hidden until fully supported
+	cmd.Flags().Bool(flagInstallDevelopment, false, "Install an isolated Elastic Agent for development. Allows a non-development agent to be installed already. (experimental)")
+	_ = cmd.Flags().MarkHidden(flagInstallDevelopment) // For internal use only.
```

Comment:
"Two agents ought to be enough for anybody."

If I expose `--namespace` then I have to go off and test it, and I want to keep the scope of what this lets us do limited to what we need while letting us expand it as needed in the future.

I'm not strictly opposed to exposing this and hiding it, but I'd prefer it be done as a follow up. Now that ON week is over I have a lot less time to consistently dedicate to this and it's useful in its current form.

Reply:
I agree that two agents ought to be enough for anybody. I just don't agree that forcing the name to `development` as that solution. I can see a case in production environments that would require two Elastic Agent's and I know they would not want one of them to be named `development`.

Not a blocker, can possibly expose `--namespace` in a follow up, if we end up supporting for real deployments.

Reply:
I added `--namespace`, but did not document it. In the integration tests I alternately use --namespace and --develop.

I like --develop as a shorthand, I can also mirror it to the run command where --namespace makes no sense but we still need some special setup to avoid needing extra configuration when running two agents on the same machine.